requirements_testing.txt

```diff
@@ -0,0 +1,11 @@
+# These are requirements specifically for building and testing JAX.
+# The requirements for running it are in build/setup.py.
+opt-einsum
```

Comment:
I'm confused by this requirement. You shouldn't need `opt_einsum` to build JAX. And if you want it to test JAX, JAX itself depends on it so you don't need to specify it here.